src/index.ts

```diff
@@ -164,57 +226,4 @@ class Main {
 
 const tsParticles = new Main();
 
-Object.freeze(tsParticles);
-
-export { tsParticles };
-
-/**
- * The new singleton, replacing the old particlesJS
- */
-window.tsParticles = tsParticles;
-
-Object.freeze(window.tsParticles);
-
-/* particles.js compatibility */
-
-/**
- * Loads the provided options to create a [[Container]] object.
- * @deprecated this method is obsolete, please use the new tsParticles.load
- * @param tagId the particles container element id
- * @param params the options object to initialize the [[Container]]
- */
-window.particlesJS = (tagId: string, params: RecursivePartial<IOptions>) => {
-    tsParticles.load(tagId, params);
-};
-
-/**
- * Loads the provided json with a GET request. The content will be used to create a [[Container]] object.
- * @deprecated this method is obsolete, please use the new tsParticles.loadJSON
- * @param tagId the particles container element id
- * @param pathConfigJson the json path to use in the GET request
- * @param callback the function called after the [[Container]] object is loaded that will be passed as a parameter
- */
-window.particlesJS.load = (tagId: string, pathConfigJson: string, callback: (container: Container) => void) => {
-    tsParticles.loadJSON(tagId, pathConfigJson).then((container) => {
-        if (container) {
-            callback(container);
-        }
-    });
-};
-
-/**
- * Adds an additional click handler to all the loaded [[Container]] objects.
- * @deprecated this method is obsolete, please use the new tsParticles.setOnClickHandler
- * @param callback the function called after the click event is fired
- */
-window.particlesJS.setOnClickHandler = (callback: EventListenerOrEventListenerObject) => {
-    tsParticles.setOnClickHandler(callback);
-};
-
-/**
- * All the [[Container]] objects loaded
- * @deprecated this method is obsolete, please use the new tsParticles.dom
- */
-window.pJSDom = () => {
-    return window.tsParticles.dom();
-};
+export { tsParticles };
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [file should end with a newline](https://app.codacy.com/manual/ar3s/tsparticles/pullRequest?prid=5308283)